storage/pv-to-path-mapping

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# map PVS to storage paths
+printf "%-40s\t%-40s\t%s\n" "PATH" "PV" "CLAIM"
+PVS=$(oc get pv -o jsonpath='{.items[*].metadata.name}' | tr ' ' "\n")
+declare -A PATH_TO_PV_MAP
+while read pv <&3; do
+  path=$(oc get pv ${pv} -o jsonpath='{.spec..path}')
```

Comment:
should it be a double dot?

Reply:
the double dot is recursive so don't have to specify glusterfs so could work for other specs in theory.